src/stdio-bridge/stdio-bridge.c

```diff
@@ -82,11 +87,30 @@ static int parse_argv(int argc, char *argv[]) {
                 case '?':
                         return -EINVAL;
 
-                case 'p':
-                        arg_bus_path = optarg;
+                case 'p': {
+                        char *e = NULL;
+
+                        e = bus_address_escape(optarg);
+                        if (!e)
+                                return log_oom();
+
+                        free_and_replace(arg_bus_path, e);
+
                         break;
 
-                default:
+                } case ARG_MACHINE: {
+                        char *e = NULL;
+
+                        e = bus_address_escape(optarg);
```

Comment:
Hmm, `bus_set_address_system_machine()` calls `bus_address_escape()`. So it seems not necessary to call this here.